dispatcher/static.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = function StaticDispatcher(dispatcher) {
```

Comment:
It would be nice to have a one-liner that describes for what a static dispatcher could be used.